src/app/components/Toolbar/Features/Contacts/ContactAccountForm.tsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from 'react'
+import { useSelector } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+import { Box } from 'grommet/es6/components/Box'
+import { Button } from 'grommet/es6/components/Button'
+import { Form } from 'grommet/es6/components/Form'
+import { FormField } from 'grommet/es6/components/FormField'
+import { TextInput } from 'grommet/es6/components/TextInput'
+import { TextArea } from 'grommet/es6/components/TextArea'
+import { selectContactsList } from 'app/state/contacts/selectors'
+import { isValidAddress } from 'app/lib/helpers'
+import { contactsActions } from 'app/state/contacts'
+import { Contact } from 'app/state/contacts/types'
+import { DeleteContact } from './DeleteContact'
+
+interface ContactAccountFormProps {
+  contact?: Contact
+  deleteHandler?: (address: string) => ReturnType<typeof contactsActions.delete>
+  setLayerVisibility: (isVisible: boolean) => void
+  submitHandler: (contact: Contact) => ReturnType<typeof contactsActions.add | typeof contactsActions.update>
```

Comment:
Can you give me a short short snippet? Are we talking about prop names, or making ContactAccountForm/Delete components connected etc. ?

Reply:
Instead of:
```tsx
<ContactAccountForm setLayerVisibility={setLayerVisibility} submitHandler={submitHandler} />

<ContactAccountForm
  contact={contact}
  deleteHandler={deleteHandler}
  setLayerVisibility={setLayerVisibility}
  submitHandler={submitHandler}
/>

<DeleteContact
  deleteHandler={() => deleteHandler(contact.address)}
  setLayerVisibility={setDeleteLayerVisibility}
/>
```
do
```tsx
<ContactAccountForm
  onSave={(contract) => { submitHandler(contract); setLayerVisibility(false) }}
  onCancel={() => setLayerVisibility(false)} 
/>

<ContactAccountForm
  contact={contact}
  onSave={(contract) => { submitHandler(contract); setLayerVisibility(false) }}
  onDelete={(address) => { deleteHandler(address); setLayerVisibility(false) }}
  onCancel={() => setLayerVisibility(false)}
/>

<DeleteContact
  onDelete={() => onDelete(contact.address)}
  onCancel={() => setDeleteLayerVisibility(false)}
/>
```